integration-libs/cdc/schematics/add-cdc/index.ts

```diff
@@ -70,9 +78,9 @@ function addCdc(options: SpartacusCdcOptions): Rule {
       content: `<${CDC_CONFIG}>{
           cdc: [
             {
-              baseSite: 'electronics-spa',
-              javascriptUrl: '<url-to-cdc-script>',
-              sessionExpiration: 3600
+              baseSite: '${options.baseSite}',
+              javascriptUrl: '${jsSDKUrl}',
```

Comment:
In any case, if ever options.jsSDK is not available with defaults and such, therefore null / undefined, we can handle this easily by doing the snippet below

```suggestion
              javascriptUrl: '${options.jsSDKUrl ?? 'JS_SDK_URL_PLACEHOLDER'}',
```

Basically, make use of the nullish coalescing operator to catch undefined and null values.
This applies for baseSite and sessionExpiration.

Again, based on what I said here https://github.com/SAP/spartacus/pull/15756/files#r875015142 as it seems these config properties are required and NOT optional.

Just to repeat, if they were optional, for instance, sessionExpiration is optional, then you would have to configure the object as 

```
{
 baseSite: 'xxx',
 javaScriptUrl: 'xxx'
}
```

^ omitted sessionExpiration from the config as it is optioanl

Reply:
that is correct, all these parameters are required, but the only difference is that `baseSite` and `sessionExpiration` can take in the predefined value, but for `jsSDKUrl`, the URL must be provided by the user.

Reply:
it can be predefined from the user, but nothings stops the user from pressing enter. Therefore, as mentioned previously, we would want to place a default value in case that happens.

All you need to do in the end afterwards is to document and let your customers know what they need to do. Example this CDS docs that explains the flow of things https://sap.github.io/spartacus-docs/cds-integration/#enabling-intelligent-selling-services-for-sap-commerce-cloud-in-spartacus